mlflow/utils/search_utils.py

```diff
@@ -44,8 +46,18 @@ class SearchUtils(object):
         '!=': operator.ne,
         '<=': operator.le,
         '<': operator.lt,
+        'LIKE': re.match,
```

Comment:
This doesn't seem to be the same kind of thing as the other operators, what's going on?

Reply:
I'm not sure to understand your comment, here the operator corresponding to the LIKE operator is a regexp for the file store, we couldn't use a basic operator like equal or le.